diff --git a/index.js b/index.js
index c1bf39c362ce7ea598845ee39962173c0542b0e2..804ac30789ea527857638cc4b53408917cadec9b 100644
--- a/index.js
+++ b/index.js
@@ -1,7 +1,7 @@
 import { EOL } from 'os';
 import fs from 'fs';
 import { Plugin } from 'release-it';
-import conventionalRecommendedBump from 'conventional-recommended-bump';
+import { Bumper } from 'conventional-recommended-bump';
 import conventionalChangelog from 'conventional-changelog';
 import semver from 'semver';
 import concat from 'concat-stream';
@@ -36,7 +36,27 @@ class ConventionalChangelog extends Plugin {
     this.debug({ increment, latestVersion, isPreRelease, preReleaseId });
     this.debug('conventionalRecommendedBump', { options });
     try {
-      const result = await conventionalRecommendedBump(options, options?.parserOpts);
+      let bumper = new Bumper(process.cwd());
+      if (options.preset) bumper = bumper.loadPreset(options.preset);
+      if (options.tagPrefix || options.skipUnstable || options.clean) {
+        bumper = bumper.tag({
+          prefix: options.tagPrefix,
+          skipUnstable: options.skipUnstable,
+          clean: options.clean
+        });
+      }
+      if (options.filterReverts || options.path || options.from || options.to || options.format || options.ignore) {
+        bumper = bumper.commits({
+          ...(options.filterReverts ? { filterReverts: options.filterReverts } : {}),
+          ...(options.path ? { path: options.path } : {}),
+          ...(options.from ? { from: options.from } : {}),
+          ...(options.to ? { to: options.to } : {}),
+          ...(options.format ? { format: options.format } : {}),
+          ...(options.ignore ? { ignore: options.ignore } : {})
+        });
+      }
+
+      const result = await bumper.bump();
       this.debug({ result });
       let { releaseType } = result;
       if (increment) {
diff --git a/package.json b/package.json
index 521af4bad332a79f01dbce7c097659aec4243558..528233ee7a1cb9eff9e68bf167e1e1369ab3a7d8 100644
--- a/package.json
+++ b/package.json
@@ -35,8 +35,8 @@
   },
   "dependencies": {
     "concat-stream": "^2.0.0",
-    "conventional-changelog": "^5.1.0",
-    "conventional-recommended-bump": "^9.0.0",
+    "conventional-changelog": "^6.0.0",
+    "conventional-recommended-bump": "^10.0.0",
     "semver": "^7.5.4"
   },
   "devDependencies": {
